### PR TITLE
moved last activity of EV4 to first of EV5

### PR DESCRIPTION
**Pull request by StevenClontz**
additional context/details...

### CODE REVIEW

**Review by siwelwerd — COMMENTED** [review]
I actually think this works better in the end of section EV04.

File: linear-algebra/source/02-EV/05.ptx [hunk 1]
```diff
@@ -48,6 +48,27 @@
...
+        <li>
+            <p>
+                Yes
+            </p>
+        </li>
+        <li>
+            <p>
+                No
+            </p>
+        </li>
```
Comment (StevenClontz) [inline linear-algebra/source/02-EV/05.ptx:67]:
```suggestion
        <li>
            <p>
                Yeppers
            </p>
        </li>
        <li>
            <p>
                Nah man
            </p>
        </li>
```

Comment (StevenClontz) [reply linear-algebra/source/02-EV/05.ptx:67]: What if I do this @siwelwerd?

**Comment by StevenClontz** [comment]
actually this is garbage

**Pull request closed without merging**